src/cascadia/TerminalApp/App.cpp

```diff
@@ -196,18 +198,15 @@ namespace winrt::TerminalApp::implementation
         winrt::Windows::UI::Xaml::Documents::Hyperlink gettingStartedLink;
         winrt::Windows::UI::Xaml::Documents::Hyperlink documentationLink;
         winrt::Windows::UI::Xaml::Documents::Hyperlink releaseNotesLink;
-        winrt::hstring gettingStartedLabel = L"Getting Started\n";
-        winrt::hstring documentationLabel = L"Documentation\n";
-        winrt::hstring releaseNotesLabel = L"Release Notes\n";
         std::wstringstream aboutTextStream;
 
         gettingStarted.Text(gettingStartedLabel);
         documentation.Text(documentationLabel);
         releaseNotes.Text(releaseNotesLabel);
 
-        std::wstring gettingStartedUriValue{ L"https://github.com/microsoft/terminal/blob/master/doc/user-docs/UsingJsonSettings.md" };
-        std::wstring documentationUriValue{ L"https://github.com/microsoft/terminal/blob/master/doc/user-docs/index.md" };
-        std::wstring releaseNotesUriValue{ L"https://github.com/microsoft/terminal/releases" };
+        std::wstring gettingStartedUriValue{ L"https://aka.ms/terminal-getting-started" };

```

Comment:
URIs are usually also stored in the resource file and loaded instead of hardcoded string table constants.